vault/resource_auth_backend.go

```diff
@@ -146,6 +147,7 @@ func authBackendRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	for path, auth := range auths {
+		path = strings.TrimSuffix(path, "/")
```

Comment:
This should fix https://github.com/terraform-providers/terraform-provider-vault/issues/182